### PR TITLE
Support Go 1.11 Modules by goagen

### PR DESCRIPTION
**Pull request by tchssk**
#1914

### CODE REVIEW

File: goagen/codegen/workspace.go [hunk 1]
```diff
@@ -368,6 +406,107 @@ func PackageName(path string) (string, error) {
 	return pkgNames[0], nil
 }
 
+// Copeied from cmd/go/internal/modload/init.go.
```
Comment (raphael) [inline goagen/codegen/workspace.go]: typo

Comment (tchssk) [reply goagen/codegen/workspace.go]: L425 had a same typo. I fixed both.

**Comment by raphael** [comment]
Thanks!

**Comment by raphael** [comment]
I just realized that the tests don't pass on Windows  - see [appveyor](https://ci.appveyor.com/project/RaphaelSimon/goa-oqtis/branch/master). @tchssk do you think you could take a look? thank you!

**Comment by raphael** [comment]
Also this should probably be ported to v1.

**Comment by raphael** [comment]
Fixed in #1974

**Comment by tchssk** [comment]
Thank you for fixing the tests.

* Has #1930 been applied onto v1? It should be ported too.
* The new version contains some new features. Is it not v1.5.0 but 1.4.1?

**Comment by raphael** [comment]
hmm yes #1930 should be ported, good catch!
The only new feature really is support for Go modules I think? it didn't seem like that justified a minor version upgrade.

**Comment by raphael** [comment]
I have ported #1930 to v1 and retagged v1.4.1, thank you!

**Comment by tchssk** [comment]
OK. I agree to v1.4.1.

**Comment by tchssk** [comment]
BTW you should announce v1.4.1 on Twitter. @raphael

**Pull request merged**